projects/multiple-servers/README.md

```diff
@@ -10,13 +10,15 @@ Timebox: 5 days
 
 ## Learning objectives:
 
-- Basic microservices ideas, separating concerns of services
+- Describe microservices
+- Separate concerns of microservices
+- Replicate a local setup in the cloud on a virtual machine
 - Configure nginx to talk to 2-3 copies of the API server
-- Some web security ideas (CORS)
-- Reverse proxy configuration, routing on path
-- Health checks
+- Resolve CORS issues
+- Configure a reverse proxy, routing on path
+- Conduct server health checks
```

Comment:
I'm not 100% on this one... While we have the trainee observe nginx marking a server as bad, I don't think we have them _conduct_ health checks, and we don't really talk about how the health checks happen or how servers are marked as healthy again...

I'd maybe remove this LO, or if we want to keep it, we should maybe expand the last section a little...

Reply:
remove it for now I think
```suggestion

```